packages/common/src/repository/activeSubmissionRepository.ts

```diff
@@ -52,6 +52,50 @@ const repository = (dependencies: Dependencies) => {
 			const updated = await db.update(submissions).set(newData).where(eq(submissions.id, submissionId)).returning();
 			return updated[0];
 		},
+
+		/**
+		 * Get Active Submission by category
+		 * @param {number} categoryId Category ID
+		 * @returns An Active Submission
+		 */
+		getActiveSubmissionWithRelations: async (categoryId: number) => {
```

Comment:
Can we only have one active submission per category? I would think we could have multiple, one per user...

Reply:
this is an important point I missed. We should be able to handle multiple submissions per category at least one per user. Will add userName param.